src/database.js

```diff
@@ -10,9 +10,10 @@ async function connect() {
 		driver: sqlite3.Database
 	});
 
-	await database.exec(`CREATE TABLE IF NOT EXISTS server_settings (
+	await database.run(`CREATE TABLE IF NOT EXISTS server_settings (
```

Comment:
Was there an issue with using `exec` here? (only going to comment once, I see this is changed multiple times)

Reply:
While making the polls functionality I typed an apostrophe into a poll's title and it failed because the SQL became invalid. Using `run` fixed this issue, so I just replicated it over to the other database queries to avoid it happening elsewhere. This specific change was to standardize it with the rest of them, along with the other marked out change.